tests/phpunit-admin.xml

```diff
@@ -0,0 +1,15 @@
+<phpunit bootstrap="bootstrap-admin.php">
+    <php>
+      <server name="kernel.environment" value="test" />
```

Comment:
I've moved the environment variable configuration from the bootstrap script to the config file.